views/js/pciCreator/dev/textReaderInteraction/creator/widget/states/Question.js

```diff
@@ -0,0 +1,144 @@
+/*global define*/
+define([
+    'taoQtiItem/qtiCreator/widgets/states/factory',
+    'taoQtiItem/qtiCreator/widgets/interactions/states/Question',
+    'taoQtiItem/qtiCreator/widgets/helpers/formElement',
+    'taoQtiItem/qtiCreator/editor/simpleContentEditableElement',
+    'taoQtiItem/qtiCreator/editor/containerEditor',
+    'tpl!textReaderInteraction/creator/tpl/propertiesForm',
+    'lodash',
+    'jquery'
+], function (stateFactory, Question, formElement, simpleEditor, containerEditor, formTpl, _, $) {
+    'use strict';
+    var stateQuestion = stateFactory.extend(Question, function () {
+        var that = this,
+            $container = that.widget.$container,
+            interaction = that.widget.element;
+
+        containerEditor.create($container.find('.prompt'), {
+            change : function (text) {
+                interaction.data('prompt', text);
+                interaction.updateMarkup();
+            },
+            markup : interaction.markup,
+            markupSelector : '.prompt',
+            related : interaction
+        });
+
+        //create choise labels inline editors
+        simpleEditor.create($container, '.js-choice-label', _.noop);
+
+        //Enable page CKEditor on selected tab and disable on the rest tabs.
+        $container.on('selectpage.' + interaction.typeIdentifier, function (event, index) {
+            var editor,
+                pageIndex;
+            $container.find('.tr-page').each(function () {
+                editor = $(this).find('.container-editor').data('editor');
+                pageIndex = $(this).data('page-num');
+                if (editor) {
+                    editor.setReadOnly(pageIndex != index);
+                }
+            });
+        });
+
+        //Destroy page CKeditors when page rerenders
+        $container.on('beforerenderpages.' + interaction.typeIdentifier, function () {
+            containerEditor.destroy($container.find('.passage'));
+        });
+
+        //Init page CKeditors after render
+        $container.on('createpager.' + interaction.typeIdentifier, function () {
+            initEditors($container, interaction);
+        });
+
+        initEditors($container, interaction);
+
+        /**
+         * Function initializes the editors on the each page.
+         * @param {jQuery DOM element} $container - interaction container
+         * @param {object} interaction 
+         * @returns {undefined}
+         */
+        function initEditors($container, interaction) {
+            var $pages = $container.find('.tr-page');
+            $pages.each(function () {
+                var pageId = $(this).data('page-id'),
+                    pageIndex = $(this).data('page-num');
+
+                containerEditor.create($(this).find('.passage'), {
+                    change : function (text) {
+                        var pageData = _.find(interaction.properties.pages, function (page) {
```

Comment:
This approach will not work here because I need not strong comparison here (by default lodash use strong comparison in the `_.find()` function).